articles/azure-monitor/app/monitor-functions.md

```diff
@@ -37,6 +37,12 @@ On the function app **Overview** pane, go to **Application Insights**. Under **C
 > [!div class="mx-imgBorder"]
 :::image type="content" source="./media//functions/collection-level.jpg" lightbox="./media//functions/collection-level.jpg" alt-text="Screenshot that shows the how to enable the AppInsights Java Agent.":::
 
+### Configuration
+
+To configure this feature for an Azure Function App not on a consumption plan, add environment variables in App settings. To review available configurations, see [Configuration options: Azure Monitor Application Insights for Java](../app/java-standalone-config.md). 
+
+For Azure Functions on a consumption plan, the available configuration options are limited to WEBSITE_SITE_NAME, APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL, and APPLICATIONINSIGHTS_SELF_DIAGNOSTICS_LEVEL. To make additional configurations on a consumption plan Function, deploy your own agent, see [Distributed Tracing for Java Functions](https://github.com/Azure/azure-functions-java-worker/wiki/Distributed-Tracing-for-Java-Azure-Functions#customize-distribute-agent). Deploying your own agent will result in an 8 to 9 second cold start implication on your Function.
```

Comment:
can we explain the limitation? that will be useful.

Reply:
not sure I understand what you mean by "explain the limitation". Are you saying that we should say add something like.

 --> _For Azure Functions on a consumption plan, the available configuration options are limited to WEBSITE_SITE_NAME, APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL, and APPLICATIONINSIGHTS_SELF_DIAGNOSTICS_LEVE because we are pre-loading the Java agent prior to when the function is called._

In my view, this is to much implementation detail for the customer. Also, I will need the exact verbiage for this explanation. Let's sync at Java Team Meeting.

Reply:
let's hear Trask's feedback. i thought it's better to explain why other configuration won't work in the consumption plan. that was that customer's concern.